server/services/auditService.ts

```diff
@@ -117,7 +117,7 @@ export enum PostAction {
   EditHairTypeOrColour = 'EDIT_HAIR_TYPE_OR_COLOUR',
   EditHeight = 'EDIT_HEIGHT',
   EditMilitaryServiceInformation = 'EDIT_MILITARY_SERVICE_INFORMATION',
-  EditNameDecision = 'EDIT_NAME_CHANGE_DECISION',
+  EditNamePurpose = 'EDIT_NAME_PURPOSE',
```

Comment:
This also addresses the comment made here: https://github.com/ministryofjustice/hmpps-prisoner-profile/pull/877